proposals/dao/tip_123.ts

```diff
@@ -0,0 +1,190 @@
+  console.log(' Equity diff                            [M]e18 ', Number(eqDiff) / 1e24);
+  console.log('----------------------------------------------------');
+
+  // 0. Assert overcollaterised
+  expect(await contracts.collateralizationOracle.isOvercollateralized()).to.be.true;
+
+  // 1. Verify Fei DAO timelock admin burned
+  expect(await contracts.feiDAOTimelock.admin()).to.equal(addresses.daoTimelockBurner);
+
+  // Verify no addresses have GOVERN_ROLE, GUARDIAN_ROLE. One has PCV_CONTROLLER_ROLE, one has MINTER
+  expect(await contracts.core.getRoleMemberCount(ethers.utils.id('GOVERN_ROLE'))).to.equal(0);
+  expect(await contracts.core.getRoleMemberCount(ethers.utils.id('GUARDIAN_ROLE'))).to.equal(0);
+  expect(await contracts.core.getRoleMemberCount(ethers.utils.id('PCV_CONTROLLER_ROLE'))).to.equal(1);
+  expect(await contracts.core.getRoleMemberCount(ethers.utils.id('MINTER_ROLE'))).to.equal(1);
+
+  // 2. Verify Rari Fei deprecated timelock burned
+  expect(await contracts.rariInfraFeiTimelock.beneficiary()).to.equal(addresses.feiTimelockBurner1);
+
+  // 3. Verify Rari Tribe deprecated timelock burned
+  expect(await contracts.rariInfraTribeTimelock.beneficiary()).to.equal(addresses.tribeTimelockBurner1);
+
+  // Verify Fuse multisig does not have any delegated TRIBE
+  expect(await contracts.tribe.getCurrentVotes(addresses.fuseMultisig)).to.equal(0);
+
+  // 4. Verify Fei Labs Tribe timelock burned
+  expect(await contracts.feiLabsVestingTimelock.beneficiary()).to.equal(addresses.tribeTimelockBurner2);
+
+  // 5. Verify Tribe minter set to zero address and inflation is the minimum of 0.01% (1 basis point)
+  expect(await contracts.tribe.minter()).to.equal(ethers.constants.AddressZero);
+  expect(await contracts.tribeMinter.annualMaxInflationBasisPoints()).to.equal(1);
+
+  // 6. Verify can not queue on DAO timelock
+  await verifyCanNotQueueProposals(contracts, addresses);
+
+  // 7. Verify proxyAdmin controlled by DAO timelock
+  expect(await contracts.proxyAdmin.owner()).to.equal(addresses.feiDAOTimelock);
+
+  // 8. Verify can permissionlessly undelegated TRIBE from timelock via TRIBE timelock burner
+  const delegatee = '0x0d4ba14ca1e990654c6f9c7957b9b23f8a1429dc';
+  const expectedDelegateeDelegation = ethers.constants.WeiPerEther.mul(1_000_000);
+  expect(await contracts.feiLabsVestingTimelock.delegateAmount(delegatee)).to.equal(expectedDelegateeDelegation);
+
+  await contracts.tribeTimelockBurner2.undelegate(delegatee);
+
+  const finalTribeDelegated = await contracts.feiLabsVestingTimelock.totalDelegated();
+  const undelegatedAmount = initialTotalTribeDelegation.sub(finalTribeDelegated);
+
+  // Verify delegatee no longer has a delegation
+  expect(await contracts.feiLabsVestingTimelock.delegateAmount(delegatee)).to.equal(0);
+
+  // Verify total delegation decrease was equal to the delegates delegation
+  expect(undelegatedAmount).to.equal(expectedDelegateeDelegation);
```

Comment:
TODO: Add verification for rariFeiBurnerTimelock can be burned permissionlessly. Also add a verification for the rariTribeBurnerTimelock can be undelegated. 

For all TRIBE burner timelocks, call sendTribeToTreaury() and check works as expected